doc/080_examples.rst

```diff
@@ -353,3 +353,56 @@ system.
 
    root@a3e580b6369d:/# sudo -u restic /home/restic/bin/restic --exclude={/dev,/media,/mnt,/proc,/run,/sys,/tmp,/var/tmp} -r /tmp backup /
 
+*****************************************************
+Pulling a Backup with HTTP over a ssh tunnel 
+*****************************************************
+
+This example will show you how to pull a backup from a remote machine via http over a ssh tunnel. 
+
+Motivation
+==========
+
+If you do a backup like this, you do not need a publicly exposed server where the backup can be stored (like a sftp server).
+
+A specific use case for this could be a backup of a cloud server (e.g. VPS) to your local PC.
+
+Running a local restic rest server
+==================================
+
+Install the container image for the `rest server <https://github.com/restic/rest-server>`__:
+
+.. hint:: you can use podman or docker, both should work
+
+.. code-block:: console
+
+   docker pull restic/rest-server:latest
+
+Run container / rest server:
+
+.. code-block:: console
+
+   docker run --rm -e DISABLE_AUTHENTICATION=0 -p 2555:8000 -v /path/to/local/repo:/data --name restic_rest_server restic/rest-server
```

Comment:
I think it would make sense to also pass `--append-only` to rest-server. Then the remote server also can't wipe the backup. It only has to be removed for `forget`/`prune` operations.